bin/plugin/commands/test/fixtures/pull-requests.json

```diff
@@ -15265,5 +15265,112 @@
 		"performed_via_github_app": null
+	},
+	{
+		"url": "https://api.github.com/repos/WordPress/gutenberg/issues/XXXX",
+		"repository_url": "https://api.github.com/repos/WordPress/gutenberg",
+		"labels_url": "https://api.github.com/repos/WordPress/gutenberg/issues/XXXX/labels{/name}",
+		"comments_url": "https://api.github.com/repos/WordPress/gutenberg/issues/XXXX/comments",
+		"events_url": "https://api.github.com/repos/WordPress/gutenberg/issues/XXXX/events",
+		"html_url": "https://github.com/WordPress/gutenberg/pull/XXXX",
+		"id": 123456,
+		"node_id": "123456",
+		"number": 123456,
+		"title": "Placeholder to test that PRs by bots are removed.",
+		"user": {
+			"login": "@dependabot[bot]",
+			"id": 123456,
+			"node_id": "123456",
+			"avatar_url": "https://avatars.githubusercontent.com/u/123456?v=4",
+			"gravatar_id": "",
+			"url": "https://api.github.com/users/torounit",
+			"html_url": "https://github.com/torounit",
+			"followers_url": "https://api.github.com/users/torounit/followers",
+			"following_url": "https://api.github.com/users/torounit/following{/other_user}",
+			"gists_url": "https://api.github.com/users/torounit/gists{/gist_id}",
+			"starred_url": "https://api.github.com/users/torounit/starred{/owner}{/repo}",
+			"subscriptions_url": "https://api.github.com/users/torounit/subscriptions",
+			"organizations_url": "https://api.github.com/users/torounit/orgs",
+			"repos_url": "https://api.github.com/users/torounit/repos",
+			"events_url": "https://api.github.com/users/torounit/events{/privacy}",
+			"received_events_url": "https://api.github.com/users/torounit/received_events",
+			"type": "bot",
```

Comment:
I've manually added this entry to the fixture data to simulate a PR by a "bot" user. This allows us to verify it is excluded in the final output.

Reply:
Maybe we should keep it in a separate file and merge with the one autogenerated from an old release? The risk is that we would lose this custom entry if it gets regenerated in the future. It’s not a blocker but it’s something that we won’t have under control moving forward.

Reply:
✅ Agreed. Done.